staging/src/k8s.io/kubectl/pkg/explain/v2/templates/plaintext.tmpl

```diff
@@ -196,18 +198,37 @@ Takes dictionary as argument with keys:
     schema: openapiv3 JSON schema which contains the field
     name: name of field
     short: limit printing to a single-line summary
+    verboseRecursive: style the line and print everything
```

Comment:
Similarly to my other comment the refactoring would extend to the template where we remove references to verbose recursive. Verbose recursive mode would set `short` here to false and possibly another parameter to control the indentation styling

Reply:
Thanks @alexzielenski i'll for sure change that.